prestoadmin/yarn_slider/config.py

```diff
@@ -57,11 +64,11 @@
                     'Connection failed for %%(%s)s@%%(%s)s:%%(%s)d. ' +
                     'Re-enter connection information.'),
 
-    SingleConfigItem(DIR, 'Enter the directory to install slider into on ' +
+    SingleConfigItem(DIR, 'Enter the directory to install slider into on '
                      'the slider master:', '/opt/slider', None),
 
     MultiConfigItem([
-        SingleConfigItem(SLIDER_USER, 'Enter a user name for conducting ' +
+        SingleConfigItem(SLIDER_USER, 'Enter a user name for conducting '
```

Comment:
too many big words.  Makes it seem unnecessarily complicated.  How about 'Enter a user name for running slider'